Makefile

```diff
@@ -178,37 +143,62 @@ include build/makelib/golang.mk
 
 RELEASE_VERSION=$(VERSION)
 RELEASE_BIN_DIR=$(BIN_DIR)
-RELEASE_CLIENT_SERVER_PLATFORMS=$(CLIENT_SERVER_PLATFORMS)
-RELEASE_CLIENT_ONLY_PLATFORMS=$(CLIENT_ONLY_PLATFORMS)
```

Comment:
do we no longer have a concept of client only platforms? i didn't see that this was moved to somewhere else from a quick search.